Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM node:18-alpine AS base
+
+FROM node:20-alpine3.17 AS base
```

Comment:
This is not necessary to fix vulnerabilities now that we are on node 18. Also, as you can see in [this PR](https://github.com/grafana/grafana-image-renderer/pull/448) that updates the image renderer to use Node 18, upgrading Node is not about just the Docker image, so I think we should keep using the `node:18-alpine` base image.